db/migrate/20240902111455_change_adopter_foster_account_null_on_adopter_applications.rb

```diff
@@ -1,6 +1,10 @@
 class ChangeAdopterFosterAccountNullOnAdopterApplications < ActiveRecord::Migration[7.1]
   def change
-    remove_index :adopter_applications, column: [:adopter_foster_account_id], name: :index_adopter_applications_on_adopter_foster_account_id
-    change_column_null(:adopter_applications, :adopter_foster_account_id, true)
+    if index_exists?(:adopter_applications, :adopter_foster_account_id)
```

Comment:
I noticed building the db from migrations instead of the schema was not working because of these group of migrations. I wrapped these changes in conditionals as a quick fix instead of trying to edit our migration history.